util/shlib_wrap.sh.in

```diff
@@ -1,5 +1,25 @@
 #!/bin/sh
 
+# To test this OpenSSL version's applications against another version's
+# shared libraries, simply set
+#
+#     OPENSSL_REGRESSION=/path/to/other/OpenSSL/build/tree
+if [ -n "$OPENSSL_REGRESSION" ]; then
+    shlibwrap=$OPENSSL_REGRESSION/util/shlib_wrap.sh
+    if [ -x "$shlibwrap" ]; then
+        # We clear OPENSSL_REGRESSION to avoid a loop, should the shlib_wrap.sh
+        # we exec also support that mechanism...
+        OPENSSL_REGRESSION= exec "$shlibwrap" "$@"
+    else
+        if [ -f "$OPENSSL_REGRESSION" ]; then
```

Comment:
I suppose you meant to check -f "$shlibwrap", not $OPENSSL_REGRESSION. I also sense inconsistency in quoting. For example -n "$OPENSSL_REGRESSION" suggests that you are willing to tolerate white spaces in $OPENSSL_REGRESSION. But next line, shlibwrap won't tolerate them.

Reply:
Right.  Fixed.